experimental/generation/generator/packages/library/templates/standard/en-us/form.en-us.lg

```diff
@@ -16,8 +16,8 @@
 # propertyValue(property, val)
 - ${template(`${property}_Value`, val)}
 
-# entityValue(property, val)
-- ${template(`${property}Entity_Value`, val)}
+# entityValue(entity)
+- [${join(foreach(entity.value, val, template(`${entity.name}_Value`, val)), ', ')}]
```

Comment:
This should work for any entity value.  The join if required should be in the individual entity value, i.e. for a list entity like BreadEntity we would display just the value for 1 and the array form for multiple.  For dimension the resolution would never be an array and it would just display the value.  I think you can use the complexValue for all of these.  So this definition would just be the template part.